README.md

```diff
@@ -61,6 +61,8 @@ guarantee. If an attacker can control the regular expressions that
 will be matched against, they will be able to successfully mount a
 denial-of-service attack. Be warned.
 
+See PERFORMANCE.md for some examples.
```

Comment:
I don't think this will autolink, might be nice to add an explicit link.

Reply:
👍 done. Didn't know it was possible, but it's documented here: https://github.com/blog/1395-relative-links-in-markup-files